src/libraries/Microsoft.Extensions.Configuration.Json/src/Microsoft.Extensions.Configuration.Json.csproj

```diff
@@ -8,18 +8,22 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="Microsoft.Extensions.Configuration" />
-    <Reference Include="Microsoft.Extensions.Configuration.Abstractions" />
-    <Reference Include="Microsoft.Extensions.Configuration.FileExtensions" />
-    <Reference Include="Microsoft.Extensions.FileProviders.Abstractions" />
-    <Reference Include="System.Memory" />
-    <Reference Include="System.Text.Json" />
+    <ProjectReference Include="..\..\Microsoft.Extensions.Configuration\ref\Microsoft.Extensions.Configuration.csproj" />
```

Comment:
Should we introduce a `LibrariesRootDir` or something like that to make this simpler? IIRC there was already one.

Reply:
I prefer the relative path to be honest. Relying on an msbuild property a) makes the project file harder to read and b) makes it less portable.

Reply:
That really depends on which name you use for the property, but if you choose correctly I would argue that it is much better because of the exact two reasons you pointed above. Relative is less readable as I need to know the current directory, and it is less portable in case we want to add any folders in between.

Reply:
Giving it a second thought, I agree. Mainly because the existing property `LibrariesProjectRoot` is unambiguous and conveys the meaning well. I used that variable in places that I touched and where it made sense (referencing another leaf project). Thanks